### ISSUE TITLE
Validator rejects valid UNCEFACT invoice

### ISSUE THREAD

**Issue by tklein** [itplr-kosit/validator-configuration-xrechnung#39]
I found a problem with the validator version 1.3.1. 

The attached XRechnung in UNCEFACT is rejected due to BR-Z-8, however it looks perfectly valid to me...

[out.xml](https://github.com/itplr-kosit/validator/files/5474859/out.txt)

It seems to me that the actual problem occurs with the charge indicator for allowances on document level in case the VAT category code is 'Z'.

The XPath expression from rule BR-Z-8 on the attached document returns 55.53, although it should return nothing.

```xpath
sum(
/rsm:CrossIndustryInvoice/rsm:SupplyChainTradeTransaction/ram:ApplicableHeaderTradeSettlement/ram:SpecifiedTradeAllowanceCharge[ram:ChargeIndicator/udt:Indicator=true() and ram:CategoryTradeTax/ram:CategoryCode='Z']/ram:ActualAmount
)
```

However the actual XML looks like this (see attached XML)

```xml
<ram:SpecifiedTradeAllowanceCharge>
            <ram:ChargeIndicator>
               <udt:Indicator>false</udt:Indicator>
            </ram:ChargeIndicator>
            <ram:BasisAmount>55.53</ram:BasisAmount>
            <ram:ActualAmount>55.53</ram:ActualAmount>
            <ram:Reason>Rechnungskorrektur</ram:Reason>
            <ram:CategoryTradeTax>
               <ram:TypeCode>VAT</ram:TypeCode>
               <ram:CategoryCode>Z</ram:CategoryCode>
               <ram:RateApplicablePercent>0</ram:RateApplicablePercent>
            </ram:CategoryTradeTax>
</ram:SpecifiedTradeAllowanceCharge>
```

The charge indicator is clearly set to false and the XPath expression matches anyway.

This results in BR-Z-08 to fire and rejecting the document.

Am I overlooking something or is this some bug in the schematron rules?

**Comment by rkottmann** [comment]
More precisely it is an issue fo the CEN SChematron Rules, hence I reported it here https://github.com/ConnectingEurope/eInvoicing-EN16931/issues/246

**Comment by phax** [comment]
The CEN issue was fixed for release 1.3.5
